api/lib/config.js

```diff
@@ -116,6 +116,7 @@ module.exports = (function() {
     features: {
       dayBeforeImproving: _getNumber(process.env.DAY_BEFORE_IMPROVING, 4),
       dayBeforeCompetenceResetV2: _getNumber(process.env.DAY_BEFORE_COMPETENCE_RESET_V2,7),
+      garAccessV2: isFeatureEnabled(process.env.GAR_ACCESS_V2),
```

Comment:
Ce qu'on fait actuellement, c'est de créer un compte avec auth. GAR même s'il existe un compte déjà reconcilié sur même INE. Ce qu'on veut faire, c'est soit de rajouter un auth. GAR sur un compte existant, soit le remplacer.
Dans les deux cas, on veut permettre l'utilisation de ce nouveau compte GAR  sur un compte existant Pix.
```suggestion
      garAccessV2: isFeatureEnabled(process.env.USE_NEW_GAR_ACCOUNT_ON_EXISTING_PIX_ACCOUNT),
```

Reply:
C'est plus complet que ça quand même, on veut ici juste retarder la création du compte utilisateur tant que la réconciliation n'a pas été faite.

Reply:
OK, mais c'est pour moi une défaite de ne pas trouver un nom plus explicite. 
AVOID_GAR_ACCOUNT_DUPLICATION ?